wpilib/wpilib/samplerobot.py

```diff
@@ -1,20 +1,22 @@
-# validated: 2016-12-22 JW e44a6e227a89 edu/wpi/first/wpilibj/SampleRobot.java
-#----------------------------------------------------------------------------
+# validated: 2017-12-22 TW 3c3a448d4726 edu/wpi/first/wpilibj/SampleRobot.java
+# ----------------------------------------------------------------------------
 # Copyright (c) FIRST 2008-2012. All Rights Reserved.
 # Open Source Software - may be modified and shared by FRC teams. The code
 # must be accompanied by the FIRST BSD license file in the root directory of
 # the project.
-#----------------------------------------------------------------------------
+# ----------------------------------------------------------------------------
```

Comment:
I will never understand why you insist on changing these lines.

Reply:
My linter does

Reply:
I'd dread to think about what your linter thinks about Sphinx doc comments then.

Reply:
Well my 'linter' is just pycharms auto format